packages/playwright-runner/src/index.ts

```diff
@@ -31,6 +31,7 @@ declare global {
   }
   interface FixtureParameters {
     browserName: 'chromium'|'firefox'|'webkit';
+    deviceName: null | string | BrowserContextOptions
```

Comment:
Device often implies the browser (iPhone would use WebKit and Pixel will use Android). We did not solve it on a library level, but can solve it on the test runner level. Imagine `browserType` property on each device descriptor upstream, then this fixture will pick the right browser.

Reply:
Then I would say create a issue upstream and exclude this change from the PR until its finally added.

Reply:
👍  then lets create a upstream issue for it?